### DISCUSSION TITLE
Is jupyterlab<4.0 dependency necessary?

### DISCUSSION THREAD

**Discussion by juanfcocontreras** [opengeos/leafmap#642] [Q&A] [answered]
jupyterlab<4.0 is a dependency in conda-forge builds. Is it necessary?

Thanks!

**Comment by giswqs** [comment, marked as answer]
ipyleaflet did not work well with JupyterLab 4.x previously. It seems the issue has been resolved. I am adding support for JupyterLab 4.x. Should be available in the next hour. 

https://github.com/jupyter-widgets/ipyleaflet/issues/1125
https://github.com/conda-forge/leafmap-feedstock/pull/122

**Reply by juanfcocontreras** [reply]
Thank you so much!!

**Reply by ZZMitch** [reply]
Hey @giswqs, it seems that leafmap still requires jupyterlab 3.5.3. 

This should be reproducable:
1. Create new env
2. mamba install -c conda-forge jupyterlab (4.0.10 is installed)
3. mamba install -c conda-forge leafmap (requires jupyerlab downgrade to 3.5.3 and traitlets downgrade to 5.9.0)

**Reply by giswqs** [reply]
Thank you for reporting. Fixing it now. Should be available in an hour. 
https://github.com/conda-forge/leafmap-feedstock/pull/126
